modules/nf-core/gatk4spark/baserecalibrator/tests/main.nf.test

```diff
@@ -60,12 +59,11 @@ nextflow_process {
         }
 
         then {
+            assert process.success
```

Comment:
Why do you take out all the `assert process.success` ?

Reply:
that way if it fails, we don't see all snapshots totally empty

Reply:
Make sense, we could implement that for all test in the future ^^